src/WebUI/Common/Tags.tsx

```diff
@@ -12,7 +12,7 @@ import "./Tags.scss";
 import React = require("react");
 
 export interface ITagsProperties extends IVssComponentProperties {
-    items: { [key: string]: string };
+    items: string[];
```

Comment:
actually we are changing in too many places, and tags our scenario is to show key=value, now caller has to change that which is little more work for the caller.

Instead, we can have an option in the prop, which says, show only values or key=value, based on it we can take a call.

now you can change only code in images.

Reply:
yes, I thought of the same approach, but then we will have to do conditional rendering in Tags component based on the prop showKeyValue.
If showKeyValue => then Utils.getPillTags(props.items).map...
else props.items.map
I wanted to avoid this repetitive code.

Reply:
Done